teps/0086-changing-the-way-result-parameters-are-stored.md

```diff
@@ -258,7 +260,7 @@ tasks:
   taskSpec: foo
   results:
   - name: out
-    byRef: true # <-- *new field*
+    type: stringReference # <-- *new type*
```

Comment:
we don't support specifying `results` in `pipelinetasks` -  https://github.com/tektoncd/pipeline/blob/566e2e3e20c0c6348e86c248310fcc9dbfe85166/pkg/apis/pipeline/v1beta1/pipeline_types.go#L161-L215

does this mean we'll be adding `results` in `pipelinetasks`? or was the intention to have this specification within the `taskspec`?

Reply:
I think the original proposal was setting the results in the wrong place. Let me update that